libsoftwarecontainer/lxc-softwarecontainer.in

```diff
@@ -56,10 +54,10 @@ create_rootfs()
         $rootfs/usr/lib \
         $rootfs/usr/lib64 \
         $rootfs/usr/sbin \
+        $rootfs/sys \
         $rootfs/tmp \
         $rootfs/gateways \
-        $rootfs/${CMAKE_INSTALL_PREFIX} \
-        ${ADDITIONAL_FOLDER_MOUNTS}"
```

Comment:
Ok, I was just considering if it was something that the integrator/user would like to have at some point.